code/helpers/vector.dm

```diff
@@ -26,11 +27,15 @@ Inputs:
 
 	yo = pixel_y offset of the target location (optional)
 */
-/vector/New(atom/movable/source, start, end, speed = 20, xo = 16, yo = 16)
+/vector/New(atom/movable/source, start, end, speed = 20, xo = 16, yo = 16, spin = TRUE)
```

Comment:
You spin me right round baby, right round, like a record baby, right round round round.